src/backend/routers/iep.test.ts

```diff
@@ -51,7 +66,7 @@ test("basic flow - add/get goals, subgoals, tasks", async (t) => {
   const subgoal2Id = subgoal2!.subgoal_id;
 
   await trpc.iep.addTask.mutate({
-    subgoal_id: subgoal2Id,
```

Comment:
This subgoal_id is the same as the one for the assignTask on line 60 / 75. Therefore we had to add a new subgoal (subgoal1) for this to continue to work and not cause a uniqueness violation once the database migration is run